### PR TITLE
add `dapr init` proxy issue to Common Issue

### PR DESCRIPTION
**Pull request by Lichmaker**
Signed-off-by: wuguozhang <wuguozhang@cyg.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)

## Description

Add documentation for Dapr CLI `darp init` proxy issue.

https://github.com/dapr/cli/issues/1043
https://github.com/dapr/cli/pull/1048
https://github.com/dapr/cli/pull/1049

### CODE REVIEW

**Review by mukundansundar — APPROVED** [review]
LGTM. @Lichmaker just one quick question. For proxies with auth, how is that supported here?

**Comment by mukundansundar** [comment]
@dapr/approvers-docs This should be ported to 1.9 as well ...

**Comment by Lichmaker** [comment]
> LGTM. @Lichmaker just one quick question. For proxies with auth, how is that supported here?

hummmmmm. Normally, just adding the username and password in front of the host would be fine.
For example: 
```
export HTTPS_PROXY=http://username:password@host:port
```
But I looked at the source code of the `net/http` package and couldn't find any handling of authentication. 
Maybe it is not supported at the moment. I'll check it out when I have time. Maybe there is another way to fix it. 🤔

File: daprdocs/content/en/operations/troubleshooting/common_issues.md [hunk 1]
```diff
@@ -264,3 +264,15 @@ kubectl config get-users
 ```
 
 You may learn more about webhooks [here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+## How to use the proxy when installing `darp runtime` via `darp init`
```
Comment (msfussell) [inline daprdocs/content/en/operations/troubleshooting/common_issues.md:268]:
```suggestion
## How to use the proxy when installing `dapr` via `dapr init`
```

**Comment by greenie-msft** [comment]
@pravinpushkar - Can you help explain when this ENV var is used? Which flags with Dapr init use this env var? 

This doc should be added to the dapr init docs for when a http proxy is required. Examples would be good too:
https://docs.dapr.io/reference/cli/dapr-init/

Also, we need to add this environment variable to the Environment variables page: https://docs.dapr.io/reference/environment/

**Comment by msfussell** [comment]
@mukundansundar @pravinpushkar - See comment above from @greenie-msft

**Pull request closed without merging**